pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/when/impl/AnyOfConditional.java

```diff
@@ -48,5 +52,10 @@ public AnyOfConditional(List<DeclarativeStageConditional<? extends DeclarativeSt
         public int getAllowedChildrenCount() {
             return -1;
         }
+
+        @Override
+        public Expression transformToRuntimeAST(@CheckForNull ModelASTWhenContent original) {
```

Comment:
`ASTParserUtils#transformWhenContentToRuntimeAST(ModelASTWhenContent)` is in `pipeline-model-definition`, so it's not available to `pipeline-model-extensions`. But I do need to think about that some.